homework27/src/ba/bitcamp/homework/tasks/Task1.java

```diff
@@ -0,0 +1,120 @@
+package ba.bitcamp.homework.tasks;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.concurrent.LinkedBlockingQueue;
+
+/**
+ * This class is used for counting first letter from file using consumer
+ * producer metodology
+ * 
+ *
+ *
+ */
+public class Task1 {
+    //Declaring global variables
```

Comment:
For this to be producer/consumer approach,you would have to do the following things:

- Have one thread that reads the file and enqueues jobs for processing
- Have one or more threads that consume jobs and do the needed checks

All threads must be executed in parallel.